ush/ufsda/yamltools.py

```diff
@@ -93,6 +98,33 @@ def atmanl_case(config):
     return config
 
 
+def landanl_case(config):
```

Comment:
is this any different than the atm case? Is there a way we can reuse one?

Reply:
The only difference is at the end section to call `fv3land_geom_dict` as below: 

```
config['GEOM_BKG'] = fv3land_geom_dict(case, levs, ntiles, layout, io_layout)
config['GEOM_ANL'] = fv3land_geom_dict(case_anl, levs, ntiles, layout, io_layout)
```
There are many differences in the `fv3land_geom_dict` compared with `fv3_geom_dict`. The reason I added the `landanl_case` is to show the difference in name for land analysis from the `atmanl_case` for atm analysis. If you want me to reuse the `atmanl_case` for the land, I will make the corresponding changes. Thanks.

Reply:
Yeah I think we rename it to 'fv3anl_case' or something like that and just add the differing `if` statement, that way we aren't duplicating 80% of the routine again. I think this change will then cover atm, land, and aero with only minor additions needed for their specific tasks.

Reply:
As suggested to reduce the duplications, I renamed the `atmanl_case` to `fv3anl_case`, removed the `landanl_case`, and added the `landanl_case` into the `fv3anl_case`.